ocaml/xapi/quicktest_vdi_copy.ml

```diff
@@ -51,6 +51,17 @@ let read_from_vdi ~session_id ~vdi f =
       Http.Get uri in
   http req (fun (_, fd) -> f fd)
 
+(* Simpler alternative to Xapi_vdi.wait_for_vbds_to_be_unplugged_and_destroyed,
+   using sleep instead of the event system. *)
+let wait_for_no_vbds_then_destroy ~rpc ~session_id self =
+  let wait_for_no_vbds () =
+    let start = Mtime_clock.counter () in
+    let over () = (Mtime_clock.count start |> Mtime.Span.to_s) > 4.0 in
+    while not (over ()) && Client.VDI.get_VBDs ~rpc ~session_id ~self <> [] do Unix.sleep 1 done
```

Comment:
Could you use `Unix.sleepf` here and a smaller interval like `0.1`?

Reply:
Done.